README.md

```diff
@@ -36,10 +36,20 @@ npm run dist
 
 to compile, and copy all site files to the `dist/` folder
 
+## How I should install correct node version, npm and bower?
```

Comment:
what do you think about changing from `How I should` to `How should I`?

Reply:
Changed